pages/national_resources/se_resources.md

```diff
@@ -56,29 +72,71 @@ national_resources:
-      Your_tasks: [sensitive, existing_data, data_publication]
+      Your_tasks: [sensitive, existing_data, gdpr_compliance, data_publication]
     url: https://fega.nbis.se/
+  - name: Swedish Reference Genome Portal
+    description: A web platform for aggregating, sharing, and visualising non-human eukaryotic genome assemblies and genome annotations (co-)produced by researchers affiliated with Swedish institutions.
+    related_pages:
+      Your_domain: [biodiversity]
+      Your_tasks: [existing_data, data_publication]
+    url: https://genomes.scilifelab.se/
+  - name: SciLifeLab Serve
+    description: A platform offering machine learning model serving, app hosting, and other web tools.
+    how_to_access: Available to life science researchers affiliated with a Swedish research institute
+    url: https://serve.scilifelab.se/
+  - name: SciLifeLab Precision Medicine Portal
+    description: The PM Portal is a service for researchers in the precision medicine field, designed to support and accelerate data-driven life science research in Sweden. It provides links to various data sources, customised dashboards, and resources for navigating data management challenges. Researchers can also find guidance on handling sensitive data and links to relevant tools and services.
+    how_to_access: Available to life science researchers affiliated with a Swedish research institute
+    related_pages:
+      Your_domain: [health_data, human_data]
+      Your_tasks: [existing_data, gdpr_compliance]
+    url: https://precision-medicine-portal.scilifelab.se/
+  - name: Swedish Biodiversity Data Infrastructure - SBDI
+    description: The Swedish Biodiversity Data Infrastructure (SBDI) is a national research infrastructure that provides access to biodiversity data and related services for researchers in Sweden. SBDI aims to facilitate the use of biodiversity data for research, conservation, and sustainable development.
+    how_to_access: Available to everyone.
+    related_pages:
+      Your_domain: [biodiversity]
+      Your_tasks: [existing_data, data_publication]
+    url: https://biodiversitydata.se/
+  - name: SciLifeLab Training Portal
+    description: A portal for finding training events and materials related to life science research data management and bioinformatics in Sweden. Herein, the life science community in Sweden can find opportunities to gain relevant skills, and experts can find support in creating training that captures their knowledge
+    url: https://training.scilifelab.se/
+  - name: AIDA Data Hub
+    description: AIDA Data Hub is a national infrastructure for storing and sharing research data in Sweden, with a focus on clinical innovation in data driven precision health, with a base in multi-modal diagnostics and medical imaging AI. It provides secure storage and data management services to researchers, industry, and caregivers of national significance, ensuring compliance with data protection regulations.
```

Comment:
I think description for AIDA Data Hub should be revised and rephrased to avoid plagiarism/duplication.